web/.env.example

```
@@ -0,0 +1,2 @@
+# NX_API_GATEWAY=http://[ls-hostname]/api/dm
+# NX_LS_ACCESS_TOKEN=[your LS token here]
```

since its an example file its probably fine to have it uncommented
```suggestion
NX_API_GATEWAY=http://[ls-hostname]/api/dm
NX_LS_ACCESS_TOKEN=[your LS token here]
```